modules/remove.js

```diff
@@ -0,0 +1,47 @@
+const remove = () => {
+  const idDiv = document.getElementsByTagName('button');
+  const buttonPressed = (e) => {
+    const Data = JSON.parse(localStorage.getItem('collection'));
+
+    const elementId = e.target.id;
+    document.getElementById(elementId).style.display = 'none';
+    Data.splice(elementId, 1);
+    localStorage.setItem('collection', JSON.stringify(Data));
+  };
```

Comment:
Please notice that your `remove` function isn't working as expected 😉. If I add 3 books:
![image](https://user-images.githubusercontent.com/42799579/201756404-74e33d48-6d0e-448b-b09a-aba759aaf3c8.png)

And try to remove them:
![image](https://user-images.githubusercontent.com/42799579/201756795-8529f7fb-a06f-4176-9167-0e76ec4ef403.png)
They are successfully removed from the list.

Although, one of them is still remaining in the local storage:
![image](https://user-images.githubusercontent.com/42799579/201756931-07bc42e4-73d2-4c00-82b8-5ec0a288d143.png)

So, if I reload the page, it appears again:
![image](https://user-images.githubusercontent.com/42799579/201756959-c653d270-bf13-40c9-89f4-92f0c1fc7767.png)

This is happening because your list item's ids aren't being updated after removing one of them 😄. To fix the bug, please update your item's ids after removing an element from the list.

### In-depth explanation:
Each one of your items has an id, created by you. Notice that the last element has the id '2'. (The first id = 0 and the second id = 1):
![image](https://user-images.githubusercontent.com/42799579/201758168-23ee4afb-e3ed-4476-aeae-ccb4f051dff6.png)

Although, when removing the first element:
![image](https://user-images.githubusercontent.com/42799579/201757675-6b2dcad1-7b5a-4018-97b3-3e2fb038cc57.png)

The elements' ids are still the same! Notice that the third item id is still 2 when it should now be 1 (it is now the second item since the first one was removed).

Reply:
This issue is well resolved! ✔️